packages/reserve-site/src/components/Amount.tsx

```diff
@@ -0,0 +1,36 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import { BreakPoints } from 'src/components/styles'
+
+interface AmountProps {
+  label: string
+  units: number
+  gridArea: string
+  context?: string
+}
+export default function Amount({ label, units, gridArea, context }: AmountProps) {
+  const display = new Intl.NumberFormat('default').format(units)
+
+  return (
+    <div title={context} css={css(amountStyle, { gridArea })}>
```

Comment:
It's tricky to set up but for the web wallet I use the emotion Babel plugin so I don't need to import anything and I can just write:
```
      <h1
        css={{
          backgroundColor: 'yellow',
        }}
      >
```

It might be worth it for you to set up the same here, will pay off in the long run I think

Reply:
yeah long run im def planning on doing that.